virt/virt-2-5-release-notes.adoc

```diff
@@ -70,8 +70,11 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 //CNV-6824 - Offline virtual machine snapshots
 
 //CNV-6825 - Smart cloning with snapshots
-
 //CNV-4089 - Cloning a disk quickly using the storage array
+// These two release note stories are combined in this single release note.
+* Clone virtual disks efficiently and quickly using smart-cloning. Smart-cloning
```

Comment:
A few things:

* Suggest starting with something like "You can now clone ..."
* I'd also suggest linking out to your content on smart-cloning.
* We recently updated the guidelines to specifically say not to hard wrap at 80 characters. Can you please update this to wrap naturally (and keep this in mind going forward). See the updated guidelines here: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#general-file-guidelines

Reply:
Thanks @bergerhoffer - (adding @adellape for re-review and possible merge):

Reworded, though I like to steer away from starting RN sentences with You more than a few times; I prefer using directive verbs. If starting with You is standard, let me know. Thanks.

I did add the link --- I didn't know adding links to content was standard, but this is great news. 

Can I ask how you can check the code (doc) in Github for hard wraps? Looking at the raw output, I could not distinguish it, so I deleted the content and re-pasted to let Atom do the soft wrap.